tests/test_metadata.py

```diff
@@ -647,6 +648,9 @@ def test_reference(
 
 
 @SKIP_IF_TW_MARK
+@pytest.mark.skipif(
+    not PODMAN_SELECTED, reason="we only need to test it for one runtime"
```

Comment:
```suggestion
    not PODMAN_SELECTED, reason="we only need to test it for podman"
```
Nitpicking: I would make the error message more explicit about the underlying reason for better readability in the error case.

Reply:
well, there is nothing specific to podman in the test, so that improved reasoning makes it imho actually worse. anyhow, I think I addressed your concern in the followup.